spec/request/service_instances_spec.rb

```diff
@@ -3322,29 +3322,80 @@ def check_filtered_instances(*instances)
 
       context 'when purge is true' do
         let(:query_params) { 'purge=true' }
-        let(:service_offering) { VCAP::CloudController::Service.make(requires: %w(route_forwarding)) }
         let(:service_plan) { VCAP::CloudController::ServicePlan.make(service: service_offering) }
         let(:instance) { VCAP::CloudController::ManagedServiceInstance.make(space: space, service_plan: service_plan) }
-        before(:each) do
-          @binding = VCAP::CloudController::ServiceBinding.make(service_instance: instance)
-          @key = VCAP::CloudController::ServiceKey.make(service_instance: instance)
-          @route = VCAP::CloudController::RouteBinding.make(service_instance: instance)
 
-          api_call.call(admin_headers)
-        end
+        context 'when broker is space scoped' do
+          let(:service_broker) { VCAP::CloudController::ServiceBroker.make(space: space) }
+          let(:service_offering) { VCAP::CloudController::Service.make(requires: %w(route_forwarding), service_broker: service_broker) }
 
-        it 'removes all associations' do
```

Comment:
I didn't mean to remove these expectations in the context 'when broker is global - as admin'; they make sense.

I meant to remove the before block in the 'when broker is space scoped' context.